src/components/App.tsx

```diff
@@ -406,25 +420,29 @@ export class App extends React.Component<AppProps, AppState> {
     return this.state.hasStatus;
   }
 
+  toggleWorspaceSplit() {
+    const workspaceSplits = this.state.workspaceSplits;
```

Comment:
generally I prefer when state change methods are idempotent when possible. Let's in follow-up changes make separate methods to create split / restore single pane?

Reply:
added a todo